scripts/sync-to-aws-homebrew-tap

```diff
@@ -178,6 +181,7 @@ cat << EOM > "${BREW_CONFIG_DIR}/${BINARY_BASE}.json"
     "bottle": {
         "root_url": "${BASE_ASSET_URL}",
         "sha256": {
+            "arm64_big_sur": "${MAC_ARM64_HASH}",
```

Comment:
Does this mean it only works on Big Sur OS and we need to update on new releases? Or is this how it's named in homebrew repo?

Reply:
this is how it's named in the homebrew core taps, so we decided to use the same convention. Currently, this will work for any arm64 mac though (not actually sure what the min OS is on an m1, probably actually is Big Sur